bindings/java/src/org/sleuthkit/datamodel/BlackboardArtifact.java

```diff
@@ -962,6 +962,7 @@ public int hashCode() {
 		/**
 		 * A GPS track point (geolocation data).
 		 */
+		@Deprecated
```

Comment:
Please also add a deprecated tag to the doc with a "use this instead" clause. E.g., you will find this in the docs for an old Case method: 

@deprecated Use closeCurrentCase instead.